### DISCUSSION TITLE
Example or Paper for Weighted GCN Conv

### DISCUSSION THREAD

**Discussion by naman-ali** [pyg-team/pytorch_geometric#2583] [Q&A]
Hey,

I know that GCNConv supports weighted graphs too, but my questions is that: 

**Is there any paper, reference, or example that has used GCN with a weighted adjacency matrix?**

In my case, GCN with weighted graph does not work and it is not correct even theoretically. If you provide some papers, or real examples with implementation I would appreciate it.

Thank you.

**Comment by rusty1s** [comment]
I'm not aware of any, sorry :(

**Reply by naman-ali** [reply]
No worries, thank you :)
